vendor/brave-ios/tests/BUILD.gn

```diff
@@ -0,0 +1,51 @@
+import("//build/config/ios/rules.gni")
+
+config("test_config") {
+  include_dirs = [ "tests" ]
+  out_dir = rebase_path(root_out_dir)
+  # Look for frameworks in `root_out_dir` and add a runtime search path
+  # to the framework's directory
+  ldflags = [
+    "-F$out_dir",
+    "-B@rpath/Frameworks",
+  ]
+}
+
+ios_xctest_test("brave-rewards-ios-tests") {
+  testonly = true
+
+  deps = [
+    "//brave/vendor/brave-ios:brave-rewards-ios-framework",
+    ":embed-framework",
+  ]
+
+  configs += [ "//brave/vendor/brave-ios:internal_config" ]
+  configs += [ ":test_config" ]
+  configs += [ "//build/config/compiler:enable_arc" ]
+
+  libs = [
+    "CoreData.framework",
+    "BraveRewards.framework"
```

Comment:
this shouldn't be required with `"//brave/vendor/brave-ios:brave-rewards-ios-framework",` as a dep

Reply:
I think you might need `//brave/vendor/brave-ios:brave-rewards-ios-framework+link` and then you shouldn't need https://github.com/brave/brave-core/pull/2667/files#diff-c58197a5c1b63ed5961e7036ec1c6a3eR8

Reply:
and then I think also `bundle_deps = [ "//brave/vendor/brave-ios:brave-rewards-ios-framework+bundle" ]`